README.md

```diff
@@ -51,10 +51,10 @@ This roster is seeded with all approvers from Eventing workgroups.
 | v0.22   | 2021-04-06   | markusthoemmes | evankanderson   | 2021-02-24  | 2021-03-30 |
 | v0.23   | 2021-05-18   | tcnghia        | vaikas          | 2021-04-07  | 2021-05-11 |
 | v0.24   | 2021-06-29   | dprotaso       | matzew          | 2021-05-19  | 2021-06-22 |
-| v0.25   | 2021-08-10   | yanweiguo      | ...             | 2021-06-30  | 2021-08-03 |
-| v0.26   | 2021-09-21   | ZhiminXiang    | ...             | 2021-08-11  | 2021-09-14 |
-| v0.27   | 2021-11-02   | julz           | ...             | 2021-09-22  | 2021-10-26 |
-| v0.28   | 2021-12-14   | nak3           | ...             | 2021-11-03  | 2021-12-07 |
+| v0.25   | 2021-08-10   | yanweiguo      | n3wscott        | 2021-06-30  | 2021-08-03 |
```

Comment:
@lionelvillard mind if we swap? I am gonna be on vacation during the next release

Want to add @slinkydeveloper and @matzew to the rotation?

Reply:
I left our slinky, b/c of https://github.com/knative/eventing/pull/5538#discussion_r655509290

and I am after vaikas, but the table ended here, so I stopped at the given format